plugins.d/alarm-notify.sh

```diff
@@ -430,6 +454,42 @@ send_pushover() {
     return 1
 }
 
+# -----------------------------------------------------------------------------
+# pushbullet sender
+
+send_pushbullet() {
+    local userapikey="${1}" recipients="${2}" message="${3}" title="${4}" httpcode sent=0 user
+
+    if [ "${SEND_PUSHBULLET}" = "YES" -a ! -z "${userapikey}" -a ! -z "${recipients}" -a ! -z "${message}" -a ! -z "${title}" ]
+        then
+        #https://docs.pushbullet.com/#create-push
+        for user in ${recipients}
+        do
+            httpcode=$(${curl} --write-out %{http_code} --silent --output /dev/null \
+                 -u "$userapikey": \
```

Comment:
I see at the pushbullet documentation, that the API Key should be given as `--header 'Access-Token: <your_access_token_here>'`.

This is a complete example from the documentation:

```sh
curl --header 'Access-Token: <your_access_token_here>' \
     --header 'Content-Type: application/json' \
     --data-binary '{"body":"Space Elevator, Mars Hyperloop, Space Model S (Model Space?)","title":"Space Travel Ideas","type":"note"}' \
     --request POST \
     https://api.pushbullet.com/v2/pushes
```

I understand that `email`, `body` and `title` can be given as POST parameters, but I don't see anything about an access token passed as the `user` parameter.

So, I trust it works the way you did it, This is just a warning that there might be something wrong, or unsupported in future releases of the pushbullet API.